script/delete_weird_company.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../config/environment',  __FILE__)
+
+weird_company_wql = {
+  "not"=>{"type"=>"Company"},
+  "left_plus"=>[
+    {"type"=>"Metric"},
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.